### PR TITLE
black 20.8b1

### PR DESCRIPTION
**Pull request by mariosnikolaou**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/black.rb [hunk 1]
```diff
@@ -3,9 +3,10 @@ class Black < Formula
 
   desc "Python code formatter"
   homepage "https://black.readthedocs.io/en/stable/"
-  url "https://files.pythonhosted.org/packages/b0/dc/ecd83b973fb7b82c34d828aad621a6e5865764d52375b8ac1d7a45e23c8d/black-19.10b0.tar.gz"
-  sha256 "c2edb73a08e9e0e6f65a0e6af18b059b8b1cdd5bef997d7a0b181df93dc81539"
-  revision 3
+  url "https://files.pythonhosted.org/packages/dc/7b/5a6bbe89de849f28d7c109f5ea87b65afa5124ad615f3419e71beb29dc96/black-20.8b1.tar.gz"
+  sha256 "1c02557aa099101b9d21496f8a914e9ed2222ef70336404eeeac8edba836fbea"
+  license "MIT"
+  revision 4
```
Comment (bayandin) [inline Formula/black.rb:9]:
```suggestion
```

**Review by bayandin — CHANGES_REQUESTED** [review]

File: Formula/black.rb (hunk 1 above)
Comment (mariosnikolaou) [reply Formula/black.rb]: Done.

**Comment by bayandin** [comment]
Could you please also update resources and squash all the commits into a single one with message `black 20.8b1`

**Comment by mariosnikolaou** [comment]
Updated resources and squashed commits. PTAL, thanks.

**Review by dawidd6 — APPROVED** [review]

**Review by bayandin — APPROVED** [review]
Thank you @mariosnikolaou!

**Pull request closed without merging**